.github/workflows/integrity-check-package-versions-validate.yml

```diff
@@ -0,0 +1,60 @@
+name:  Check integrity of Mirror repo: Package Versions Validate
+
+on:
```

Comment:
This workflow is fine, but it should probably run against PRs that change the integrity check. The goal should be to see this check run against this PR.